src/Catalog/Helpers/Utils.cs

```diff
@@ -35,6 +35,19 @@ public static class Utils
 
         private static readonly ConcurrentDictionary<string, string> _resourceStringCache = new ConcurrentDictionary<string, string>();
 
+        private static readonly char[] TagTrimChars = { ',', ' ', '\t', '|', ';' };
+
+        public static string[] SplitTags(string original)
+        {
+            var fields = original
+                .Split(TagTrimChars)
+                .Select(w => w.Trim(TagTrimChars))
+                .Where(w => w.Length > 0)
```

Comment:
Wouldn't `original.Split(TagTrimChars, StringSplitOptions.RemoveEmptyEntries)` do the same as the three lines above?

Reply:
No, because the `Trim` happens after the split.